.eslintrc.js

```diff
@@ -73,6 +73,14 @@ module.exports = {
     'valid-jsdoc': 0,
     'arrow-parens': 0,
   },
+  overrides: [
+    {
```

Comment:
can you add a comment to this block so it's clearer what it's for?

Reply:
narrowing seems to make it self documenting